src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/scenario/JiraSoftwareScenario.kt

```diff
@@ -1,4 +1,4 @@
-package com.atlassian.performance.tools.jirasoftwareactions.api
+package com.atlassian.performance.tools.jirasoftwareactions.api.scenario
```

Comment:
This effectively deletes the `com.atlassian.performance.tools.jirasoftwareactions.api.JiraSoftwareScenario` class, which breaks source and binary compat.

- [ ] don't break JSS